start.sh

```diff
@@ -18,15 +18,101 @@ usage() {
     echo 'that command, for testing purposes.'
 }
 
+tritonConfigured() {
+
+    # is node-triton installed?
+    local triton_installed=0
+    command -v triton >/dev/null 2>&1 && triton_installed=1
+    if [ "0" = "$triton_installed" ]; then
+        echo
+        echo "Error!"
+        echo "The Triton CLI tool does not appear to be installed!"
+        echo
+        echo "Please visit:"
+        echo "https://www.joyent.com/blog/introducing-the-triton-command-line-tool"
+        echo "for installation instructions."
+        echo
+
+        sleep 3
+        command -v open >/dev/null 2>&1 && `open https://www.joyent.com/blog/introducing-the-triton-command-line-tool` || true
```

Comment:
If the goal is to make it so that it's obvious there's an error here, what about doing something like:

```bash
echo "\e[1;31mWow big red error here.\e[0m"
```

so that it stands out in red, rather than hijacking the user's browser window.